tests/events/test_event_handlers.py

```diff
@@ -1,10 +1,20 @@
+from datetime import datetime, timezone
+
 from cccat import dto
 from cccat.events.event_handlers import handle_cat_created
 
+UTC = timezone.utc
+
 
 def test_handle_cat_created() -> None:
     data: dto.JSON = {
         "cat_id": "000000000000000000000101",
+        "partial_update_cat": {
+            "name": "Sammybridge Cat",
+            "ctime": datetime(2020, 1, 1, 0, 0, tzinfo=UTC),
+            "mtime": datetime(2020, 1, 1, 0, 0, tzinfo=UTC),
+            "url": "http://placekitten.com/200/300",
```

Comment:
[question] Should the party emitting an event be aware of what the URL should be, and why/why not?
What would happen if we add the fields `mtime` and `ctime` to payload?

Reply:
@Chandan-Baba Passing all these kinds of metadata means:
1. the published needs to be aware of what the consumer expects (which is actually undoing why we use pub-sub in the first place)
2. the `ctime` is creation time, which should never ever be updated.
3. `mtime` should be decided when the update is actually happening, not when this event is emitted. Remember, messages can be consumed in any order and at any time because these are asynchronous.